torch-build.sh

```diff
@@ -1,16 +1,24 @@
 #!/bin/bash
+set -e
 
-cd ~/git
+# conda and the env vars are set correctly in pytorch-build.py
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source $SCRIPT_DIR/pytorch-build.sh $@
 
-source torch-common.sh
-pip uninstall -y torch functorch torchvision torchtext torchaudio torchdata
+pip uninstall -y torchvision torchtext torchaudio torchdata torchbenchmark
+
+PKGS=(data vision text audio benchmark)
```

Comment:
Torchbench doesn't have a `setup.py`
```suggestion
PKGS=(data vision text audio)
```